nifi-commons/nifi-expression-language/src/test/java/org/apache/nifi/attribute/expression/language/TestQuery.java

```diff
@@ -233,6 +233,24 @@ public void testEmbeddedExpressionsAndQuotes() {
     }
 
     @Test
+    public void testJsonPath() {
+        final Map<String, String> attributes = new HashMap<>();
+        attributes.put("json",
```

Comment:
Would recommend we pull the JSON out into a file in src/test/resources. Having this live in the codebase makes it very difficult to read & edit